apps/portals/src/AppInitializer.tsx

```diff
@@ -227,87 +122,28 @@ function AppInitializer(props: { children?: React.ReactNode }) {
 
     window.addEventListener('click', updateSynapseCallbackCookie)
 
-    // Technically, the AppInitializer is only mounted once during the portal app lifecycle.
-    // But it's best practice to clean up the global listener on component unmount.
-
+    // Clean up the global listener on component unmount.
     return () => {
       window.removeEventListener('click', updateSynapseCallbackCookie)
     }
```

Comment:
In React strict mode, it will probably render twice, so this "technicality" is misleading at best.